### DISCUSSION TITLE
How we can backup our workflows(DAG)?

### DISCUSSION THREAD

**Discussion by suraj7212** [apache/dolphinscheduler#13544] [General]
We want to take backup of my workflows automatically.
Is it possible? If yes, How?
Please guide us

**Comment by fuchanghai** [comment]
do you want to release history version? 
![image](https://user-images.githubusercontent.com/33984497/223373540-93600615-68b2-4f43-a033-fbf01f59c814.png)

**Comment by davidzollo** [comment]
you can export your workflow definition through this operation. 
<img width="1435" alt="image" src="https://user-images.githubusercontent.com/15833811/223385816-58e3e5b8-b09b-435f-b599-33edd7dcd1f6.png">
